config/database.yml

```diff
@@ -13,6 +13,8 @@ default: &default
   adapter: mysql2
   encoding: utf8
   pool: 5
+  host: 127.0.0.1
```

Comment:
Can we add this as DBHOST too? It will be very useful to test the docker container in local for local development

Reply:
I have added this, not sure how these config files get loaded during deployment, I assume deployment scripts set the host and port themselves for UAT, training and prod?

Reply:
On deployment we upload the real config files in the server, but in development environment we use the same config it is in the source code. By having it in a Env variable is easier to start a docker container for development an specify some configuration for it, like:
```DBHOST=myotherdatabase docker run -it saex1:develop bash```